src/main/kotlin/com/jay/racingcar/view/InputView.kt

```diff
@@ -0,0 +1,17 @@
+package com.jay.racingcar.view
+
+class InputView {
+    companion object {
```

Comment:
```suggestion
object InputView {
```
클래스 내에 변수 및 함수를 정의할 것이 없다면, object를 활용해보는 것을 고려해보세요 😄 (이하 관련 내용 동일)

Reply:
오 수정해보겠습니다!